data_steward/resource_files/schemas/rdr/pid_rid_mapping.json

```diff
@@ -10,5 +10,11 @@
     "name": "research_id",
     "mode": "nullable",
     "description": "Research ID to replace the person_id in de-identified datasets"
+  },
+  {
+    "type": "integer",
+    "name": "vibrent_id",
+    "mode": "nullable",
+    "description": "External ID used by Vibrent, might be used to replace the person_id in fitbit de-identified datasets"
```

Comment:
Got a little more clarity on this today.  This is used to map the fitbit data to a `person_id`.  There is a mapping of
`person_id` -> `research_id` and `person_id` -> `vibrent_id`.  fitbit data is ingested with a `vibrent_id`.  This is needed to map `vibrent_id` -> `person_id` when we ingest fitbit data.

Reply:
I would suggest clarifying this in the description field.  This field is used to map the fitbit data to the correct `person_idd`.

Reply:
updated the description!